component/table.jsx

```diff
@@ -0,0 +1,48 @@
+import { css } from '../lib/dom.js'
+
+css(`
+.container {
```

Comment:
too generic class name, do not fit the component name.
We need to be carefull with class names, generic ones should only be used for precise elements.

here we will probably have a lot of different `containers` so use something different, maybe `table-container` ?

better yet, this could be a token class I think, we could create an `overflow` class with  `auto` or `none` options and make it available as a props